pkg/controller/job/jobcontroller_test.go

```diff
@@ -220,7 +220,7 @@ func TestControllerSyncJob(t *testing.T) {
 
 	for name, tc := range testCases {
 		// job manager setup
-		clientset := clientset.NewForConfigOrDie(&restclient.Config{Host: "", ContentConfig: restclient.ContentConfig{GroupVersion: testapi.Default.GroupVersion()}})
```

Comment:
:( i think thats because Hosts was changed everywhere and then i simply removed them instead of adding Host: "" back